ui/dashboards/src/components/Variables/VariableEditor.tsx

```diff
@@ -196,7 +196,7 @@ export function VariableEditor(props: {
                   </Alert>
                 ))}
               <TableContainer>
-                <Table sx={{ minWidth: 650 }} aria-label="simple table">
+                <Table sx={{ minWidth: 650 }} aria-label="table of variables">
```

Comment:
Wondering if "simple table" was from a copy-paste or something. It doesn't really describe what it is. I'm open to a better description than "table of variables."

Reply:
seems like a straightforward enough description 2 me